packages/patternfly-4/react-core/src/components/Page/PageSection.tsx

```diff
@@ -7,15 +7,22 @@ export enum PageSectionVariants {
   light = 'light',
   dark = 'dark',
   darker = 'darker'
-};
+}
+
+export enum PageSectionTypes {
+  default = 'default',
+  nav = 'nav'
+}
 
 export interface PageSectionProps extends React.HTMLProps<HTMLDivElement> {
   /** Content rendered inside the section */
   children?: React.ReactNode;
   /** Additional classes added to the section */
   className?: string;
   /** Section background color variant */
-  variant?: 'default' | 'light' | 'dark' | 'darker' | PageSectionVariants
+  variant?: 'default' | 'light' | 'dark' | 'darker';
+  /** Section type variant */
```

Comment:
when should consumers use nav? should there be an example? maybe a little more descriptive comment?

Reply:
Agreed. There is no example in core.  @srambach  will be opening up an additional issue to add one.